easybuild/easyconfigs/w/WildMagic/WildMagic-5.17-foss-2018b.eb

```
@@ -0,0 +1,34 @@
+easyblock = "MakeCp"
+
+name = 'WildMagic'
+version = '5.17'
+
+homepage = "https://www.geometrictools.com/index.html"
+description = """Wild Magic 5.17"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': False}
+
+source_urls = ['https://www.geometrictools.com/Downloads/']
+sources = ['WildMagic5p17.zip']
+checksums = ['b2caf7aa0c5309eb40965ed26439388dc1c93eb632312b40573f83e4746a9ecf']
+
+dependencies = [
+    ('freeglut', '3.0.0'),
+]
+
+build_cmd = "make CFG=ReleaseDynamic -f makefile.wm5"
+
+files_to_copy = [
+    (['SDK/Library/ReleaseDynamic/*'], 'lib'),
+    (['SDK/Include/*'], 'include'),
+]
+
+sanity_check_paths = {
+    'files': ['lib/libWm5Core.%s' % SHLIB_EXT, 'lib/libWm5Applications.%s' % SHLIB_EXT,
+              'lib/libWm5Graphics.%s' % SHLIB_EXT, 'lib/libWm5Imagics.%s' % SHLIB_EXT,
+              'lib/libWm5Mathematics.%s' % SHLIB_EXT, 'lib/libWm5Physics.%s' % SHLIB_EXT],
```

@tstrempel this can be written a bit more compact:
```suggestion
    'files': ['lib/libWm5%s.%s' % (x, SHLIB_EXT) for x in [
        'Core', 'Applications', 'Graphics', 'Imagics', 'Mathematics', 'Physics'
    ]],
  ```